### DISCUSSION TITLE
Hook an Existing Real Time Messaging Solution into Your Project

### DISCUSSION THREAD

**Discussion by ronlinet** [cmgmyr/laravel-messenger#386] [Ideas]
I came across this PHP [real time chat](http://www.phpfreechat.net/src/1.0/). It is a standalone NOSQL realtime chat solution under MIT license.  

It doesn't require pusher or any other third party service for real time message pushing would guys consider to hook it in your solution as front end wrapper which will patch the lack of real time messaging?

**Comment by cmgmyr** [comment]
I think the ideas are pretty different, from a quick glance. I'd probably pick one tool or the other.
